common/lib/dependabot/pull_request_creator/branch_namer.rb

```diff
@@ -49,10 +49,10 @@ def strategy
               files: files,
               target_branch: target_branch,
               dependency_group: dependency_group,
+              includes_security_fixes: includes_security_fixes,
               separator: separator,
               prefix: prefix,
-              max_length: max_length,
-              includes_security_fixes: includes_security_fixes
```

Comment:
any reason for the reordering?

Reply:
Sorbet needs required properties to come before optional properties. `includes_security_fixes` is required, but doesn't have a default value.